drltrace/drltrace_frontend.cpp

```diff
@@ -136,7 +150,7 @@ configure_application(char *app_name, char **app_argv, void **inject_data,
     dr_option[0] = '\0';
 
     if (op_no_follow_children.get_value() == true)
-        _snprintf(dr_option, strlen("-no_follow_children") + 1, "-no_follow_children");
+        dr_snprintf(dr_option, MAX_DR_CMDLINE, "-no_follow_children");
```

Comment:
Use BUFFER_SIZE_ELEMENTS for safer coding -- then we won't have an overflow if someone changes the decl and forgets to change here.